Sources/Views/Collections/CollectionSection.swift

```diff
@@ -71,7 +71,7 @@ private func coreLessons(_ lessons: [Episode.Collection.Section.Lesson]) -> Node
           .class([
             Class.padding([
               .desktop: [.leftRight: 5, .top: 3, .bottom: 4],
-              .mobile: [.leftRight: 3, .topBottom: 2],
+              .mobile: [.leftRight: 3, .top: 2, .bottom: 3],
```

Comment:
it looks more symmetric to give a bit more bottom padding than top, because the top gets a little extra since headers get padding.